plugins/services/src/js/pages/__tests__/DeploymentsTab-test.js

```diff
@@ -43,7 +43,7 @@ describe('DeploymentsTab', function () {
         }
       ]
     });
-    DCOSStore.dataProcessed = true;
+    DCOSStore.serviceDataReceived = true;
```

Comment:
You have only defined a getter, so wouldn't this be ignored? Or is this just mocked?